actions/maint/maint.js

```diff
@@ -0,0 +1,92 @@
+/* ************************************************************************
+* ADOBE CONFIDENTIAL
+* ___________________
+*
+* Copyright 2023 Adobe
+* All Rights Reserved.
+*
+* NOTICE: All information contained herein is, and remains
+* the property of Adobe and its suppliers, if any. The intellectual
+* and technical concepts contained herein are proprietary to Adobe
+* and its suppliers and are protected by all applicable intellectual
+* property laws, including trade secret and copyright laws.
+* Dissemination of this information or reproduction of this material
+* is strictly forbidden unless prior written permission is obtained
+* from Adobe.
+************************************************************************* */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const filesLib = require('@adobe/aio-lib-files');
+const { getAioLogger } = require('../utils');
+const appConfig = require('../appConfig');
+const { isAuthorizedUser } = require('../sharepoint');
+const sharepointAuth = require('../sharepointAuth');
+
+const ENABLE = true;
+const logger = getAioLogger();
+
+// Maintainance functions
+async function main(args) {
```

Comment:
Very useful action. Thanks for this!

Minor: Return the response as JSON for better readability. They are currently escaped and coming in as plain text.